core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/README.md

```diff
@@ -20,13 +20,18 @@ Use the java command to run the created benchmark class with the following suppo
 #### Running the Benchmark class directly
 ```shell
 java -cp $CLASSPATH \
-com.nvidia.spark.rapids.tool.benchmarks.SingleThreadedQualToolBenchmark \
+org.apache.spark.rapids.tool.benchmarks.SingleThreadedQualToolBenchmark \
 -i 3 -w 3 -a " --output-directory output eventlogs"
 ```
+`CLASSPATH` should include the path relative to the Benchmarking class being passed. 
+Refer to the example below for the same.
+
 #### Running the Benchmark class using tools jar
 ```shell
-java -cp /home/rapids-4-spark-tools_2.12-24.06.2-SNAPSHOT.jar:$SPARK_HOME/jars/* \
+java -cp $RAPIDS_TOOLS_JAR:$SPARK_HOME/jars/* \
 org.apache.spark.rapids.tool.benchmarks.SingleThreadedQualToolBenchmark \
--i 3 -w 3 -a "/home/eventlogs"
+-i 3 -w 3 -a " $EVENT_LOGS_DIR"
```

Comment:
nit: Space should not be required 

```suggestion
-i 3 -w 3 -a "$EVENT_LOGS_DIR"
```

Reply:
Faced an issue where in cases of passing string starting with `--`. Eg- `--output-directory abc eventlogs`, scallop was ignoring the -- and taking it as `output-directory eventlogs`. Hence the extra space. This can be resolved in further iterations

Reply:
Thanks @bilalbari